src/neo/SmartContract/Native/Oracle/OracleContract.cs

```diff
@@ -34,6 +35,51 @@ public sealed class OracleContract : NativeContract
         internal OracleContract()
         {
             Manifest.Features = ContractFeatures.HasStorage;
+
+            var events = new List<ContractEventDescriptor>(Manifest.Abi.Events)
+            {
+                new ContractEventDescriptor
+                {
+                    Name = "OracleRequest",
+                    Parameters = new ContractParameterDefinition[]
+                    {
+                        new ContractParameterDefinition()
+                        {
+                            Name = "RequestId",
+                            Type = ContractParameterType.Integer
+                        },
+                        new ContractParameterDefinition()
+                        {
+                            Name = "Url",
+                            Type = ContractParameterType.String
+                        },
+                        new ContractParameterDefinition()
+                        {
+                            Name = "Filter",
+                            Type = ContractParameterType.String
+                        }
+                    }
+                },
+                new ContractEventDescriptor
+                {
+                    Name = "OracleResponse",
+                    Parameters = new ContractParameterDefinition[]
```

Comment:
Add `ResponseTx`?

Reply:
It's triggered by `ResponseTx`.

Reply:
This is triggered by the oracle contract too.